erpnext/stock/doctype/warehouse_type/warehouse_type.json

```diff
@@ -0,0 +1,39 @@
+{
+ "autoname": "Prompt",
+ "creation": "2019-05-21 15:27:06.514511",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "description"
+ ],
+ "fields": [
+  {
+   "fieldname": "description",
+   "fieldtype": "Small Text",
+   "label": "Description"
+  }
+ ],
+ "modified": "2019-05-21 15:30:06.754938",
+ "modified_by": "Administrator",
+ "module": "Stock",
+ "name": "Warehouse Type",
+ "owner": "Administrator",
+ "permissions": [
```

Comment:
Give permissions to Stock Manager

Reply:
Added permission on the basis of warehouse master

Reply:
Also give write/create permission to Stock Manager role